src/store/LastDigitStatsStore.js

```diff
@@ -1,8 +1,19 @@
-import { observable, action, computed } from 'mobx';
+import { observable, action, computed, when } from 'mobx';
 
 export default class LastDigitStatsStore {
     constructor(mainStore) {
         this.mainStore = mainStore;
+        // since last digits stats is going to be rendered in deriv-app
+        // we always keep track of the last digit stats.
+        when(() => this.context, () => {
+            this.lastSymbol = this.marketDisplayName;
+            this.updateLastDigitStats();
+            // TODO: call onMasterDataUpdate on symobl change.
+            this.mainStore.chart.feed.onMasterDataUpdate(this.onMasterDataUpdate);
```

Comment:
@aminroosta it should unsubscribe `onMasterDataUpdate` and `onMasterDataReinitialize` after unmounting the component.